### PR TITLE
Switched to exoscale/raven 0.2.0.

### PR DESCRIPTION
**Pull request by chrisglass**
A single feature (tags) is exposed via a new arity on SentrySink.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(defproject spootnik/reporter "0.1.23"
+(defproject exoscale/reporter "0.1.24"
```
Comment (chrisglass) [inline project.clj]: Note to reviewers: not strictly needed - however I could not release to the spootnik namespace.

Happy to change to whatever is most relevant.

File: src/spootnik/reporter.clj [hunk 2]
```diff
@@ -249,7 +249,14 @@
...
       (try
-        (raven/capture! raven (:dsn sentry) e)
+        (raven/capture! {:http raven} (:dsn sentry) e)
+        (catch Exception e
+          (error e "could not send capture")))))
+  (capture! [this e tags]
+    (error e "captured exception")
+    (when raven
+      (try
+        (raven/capture! {:http raven} (:dsn sentry) e tags)
```
Comment (greut) [inline src/spootnik/reporter.clj]: is this code duplication avoidable? `(capture! [this e] (capture! [this e {}])` ?

Comment (chrisglass) [reply src/spootnik/reporter.clj]: I would like to as well, but I'll do this when I can prove that it's indeed equivalent (after I have added tests).

For now I aimed at doing exactly like other cases (see for example inc! dec! and mark! in the same file).

**Comment by chrisglass** [comment]
Once landed you should be able to do:

(capture! (Exception. "Blah") {:arbitrary-tag "some value"})

If this works well I'll follow up with:

1) exposing some more raven functionality to reporter
2) cleaning up and tests (like the code duplication you mentioned)

**Pull request merged**